app/main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
Use the extend() method or += operator to add multiple values to a single list.(see checklist)

FrontendDeveloper will inherit an empty skills  list from its parent, you need to extend that list instead of creating a new one